src/MigrationTools.Clients.AzureDevops.ObjectModel/Enrichers/TfsWorkItemLinkEnricher.cs

```diff
@@ -150,7 +150,25 @@ private void CreateExternalLink(ExternalLink sourceLink, WorkItemData target)
                 target.ToWorkItem().Links.Add(el);
                 if (_save)
                 {
-                    target.SaveToAzureDevOps();
+                    try
+                    {
+                        target.SaveToAzureDevOps();
+                    }
+                    catch (Exception ex)
+                    {
+                        // Ignore this link because the TFS server didn't recognize its type (There's no point in crashing the rest of the migration due to a link)
+                        if(ex.Message.Contains("Unrecognized Resource link"))
```

Comment:
I'm not sure if there's a specific TFS error id we could check for to make this more stable?

Reply:
Unfortunately not :)